app/Http/Controllers/Widgets/AlertsController.php

```diff
@@ -45,6 +47,12 @@ class AlertsController extends WidgetController
 
     public function getView(Request $request)
     {
+        $alert_count = Alert::where('state', '!=', AlertState::RECOVERED)->get()->count();
```

Comment:
This will first fetch all items, then count them in php and it wont get updated , better to do it in javascript.

    alerts_grid.bootgrid("getTotalRowCount");

Reply:
therefore where - filter to interesting stuff ... 
i'd prefer not to manipulate title via javascript similar to other widgets which have a title manipulation

Reply:
@Jellyfrog I think the whole widget gets refreshed and re-inserted to the DOM already.

Reply:
Maybe it is only the body...

Reply:
The query is wrong either way. Remove the get()

Reply:
yeah .. Eloquent Details :-) still learning ;-)
updated it ...